README.md

```diff
@@ -2,32 +2,33 @@
 # launcher ![GitHub All Releases](https://img.shields.io/github/downloads/desktopgooseunofficial/launcher/total) [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=DesktopGooseUnofficial/launcher)](https://dependabot.com) [![Build status](https://ci.appveyor.com/api/projects/status/q2iccrkgxriosdmd?svg=true)](https://ci.appveyor.com/project/VukAnd/launcher)
 
 <p align="center">
-<img width="200" height="200" src="https://i.imgur.com/ma8iAVM.png">
+<img width="200" height="200" alt="RH Launcher Logo" src="https://i.imgur.com/ma8iAVM.png">
 </p>
 
 ---
 
-welcome to resourcehub launcher!
+Welcome to ResourceHub Launcher!
 
-the resourcehub launcher is a easy and simple way to install mods for the desktop goose!
+The ResourceHub Launcher is a easy and simple way to install mods for the Desktop Goose!
 
-want to download the launcher? feel free to [download it here](https://github.com/desktopgooseunofficial/launcher/releases/latest). or do you like living on the edge? testing out new very in development features? you can [try our nightly builds](https://github.com/DesktopGooseUnofficial/launcher-nightly/releases/latest) as well!
+Want to download the launcher? Feel free to [download it here](https://github.com/desktopgooseunofficial/launcher/releases/latest). Or do you like living on the edge? Testing out new very in development features? You can [try our nightly builds](https://github.com/DesktopGooseUnofficial/launcher-nightly/releases/latest) as well!
 
-looking for the backend? check [this github repo](https://github.com/desktopgooseunofficial/launcher-backend). 
+Looking for the backend? Check [this github repo](https://github.com/desktopgooseunofficial/launcher-backend). 
 
 ## FAQ:
 
 ### When is 2.0 coming out?
 
-soon™️
+Soon™️! Meanwhile, you can [try our nightly builds](https://github.com/DesktopGooseUnofficial/launcher-nightly/releases/latest) or [our beta builds](https://github.com/Wuzado/launcher/releases/latest). Don't forget to report the bugs you may encounter to [our bug tracker](https://github.com/DesktopGooseUnofficial/launcher/issues), so they can be quickly fixed!
```

Comment:
```suggestion
Soon™️! Meanwhile, you can [try our nightly builds](https://github.com/DesktopGooseUnofficial/launcher-nightly/releases/latest) or [our beta builds](https://github.com/DesktopGooseUnofficial/launcher/releases). Don't forget to report the bugs you may encounter to [our bug tracker](https://github.com/DesktopGooseUnofficial/launcher/issues), so they can be quickly fixed!
```
sadly don't think that method would work?

Reply:
Good catch! Just copied a link to the releases... but from the repo I was working on lol

Also 12 AM zzz